core/src/test/java/org/apache/spark/JavaJdbcRDDSuite.java

```diff
@@ -32,6 +33,7 @@
 import org.junit.Test;
 
 public class JavaJdbcRDDSuite implements Serializable {
+  private String dbName = UUID.randomUUID().toString();
```

Comment:
I remember it becomes flaky when the UUID starts with numbers (in Spark SQL), e.g.. `SQLTestUtils.withTempDatabase`. Should we better copy and paste that here?

Reply:
Oh, thank you. Sure.